### DISCUSSION TITLE
Frontend H1 tidy up

### DISCUSSION THREAD

**Discussion by nhsbsa-wicoo** [nhsbsa/nhsbsa-digital-playbook#276] [Development]
On 'Articles' pages like the following, the 'Articles' text is a `<h2>` so we're missing `<h1>`s from the page. I think the pages should have unique `<h1>`s that match the page title. Unique `<h1>`s are also better for accessibility as it gives the user more context to where they actually are in the playbook. 

We might need someone from the content design team to confirm but I'd suggest as an example having either `Design` or `Design articles` and then having the page title match so it'd be either `Design - NHSBSA Digital, Data and Technology Playbook - NHSBSA` or `Design articles - NHSBSA Digital, Data and Technology Playbook - NHSBSA`.

So currently in the following image you're on a page with the `<h2>Articles</h2>`, you click into 'Content design' which takes you to another page with the `<h2>Articles</h2>`.

![screencapture-localhost-8080-nhsbsa-digital-playbook-design-2023-12-18-15_33_21 (1)](https://github.com/nhsbsa/nhsbsa-digital-playbook/assets/52259124/8fb07de2-4ea5-454b-a514-af9ca3528ab7)

**Comment by aircwo** [comment]
I think we should update the heading level only - as an issue. Changing it from H2 to H1. 
The "Articles" text can be configured within each `index.md` file - so this would be something different for each collection of articles.

**Comment by yantantether** [comment]
I think we should leave this issue for later. 

The `articles` section is shown whenever there are articles with 'related' 11ty tags. The whole approach is really confusing, and difficult to manage as an author. I think we should throw it away and just have a simple list of related articles defined in the front matter of each page.

Furthermore, the same related articles concept is used both for showing cards in a top level navigational page (e.g. development, design) and to show things related to an actual article. I also think this is confusing. It would be more obvious to adopt a different template for top level navigational content, and then you can choose an H1 more easily.

But with all this, we're going to revisit navigation, at which time we can choose a more considered approach.
